components/nss-libs-android/README.md

```diff
@@ -0,0 +1,5 @@
+# nss-libs-android
+
+This is an android module that just provides the NSS libraries. It's separate
+solely so that it can be removed from the dependency graph by users who are
+providing their own NSS'es, e.g. anybody who uses geckoview.
```

Comment:
Oh, nice one!  I guess Fenix would just dependency-substitute this with geckoview?

Reply:
Yes, hopefully.

It also might just work out if the fenix megazord doesn't depend on it.